src/api/sso/src/index.js

```diff
@@ -15,7 +15,7 @@ const service = new Satellite({
     app.use(
       session({
         store: process.env.NODE_ENV === 'production' && new RedisStore({ client: Redis() }),
-        secret: process.env.SECRET || `telescope-has-many-secrets-${Date.now()}!`,
+        secret: process.env.JWT_SECRET || `telescope-has-many-secrets-${Date.now()}!`,
```

Comment:
Does this default secret need to be in any particular form?  `telescope-has-many-secrets-${Date.now()}!` is 41 chars btw.

Reply:
https://github.com/Seneca-CDOT/telescope/blob/0723bb957f453ad6f80e690307b711d1b548209d/config/env.development#L279

at least 32 chars long

Reply:
the old secret

https://github.com/Seneca-CDOT/telescope/blob/0723bb957f453ad6f80e690307b711d1b548209d/config/env.development#L76

Reply:
OK, so this is fine.

We'll have to figure out what our solution is going to be in staging/production for these keys.